com.unity.multiplayer.mlapi/Editor/TrackedObjectEditor.cs

```diff
@@ -23,7 +23,11 @@ public override void OnInspectorGUI()
         {
             Init();
             base.OnInspectorGUI();
-            if(NetworkingManager.Singleton != null && NetworkingManager.Singleton.IsServer)
+
+            //FIXME: Singleton Conversion, get all of them and be sure to get the server!
+            NetworkingManager manager = NetworkingManagerEditor.GetNetworkingManager(true);
```

Comment:
So I think generally in the non-singleton case I would expect the editors to show only useful data if the actual object being selected is a NetworkManager... I just don't think that picking the first thing you see if actually useful. 

Also FindObjectOfType doesn't guarantee order so you could call it more than once and get different objects which doesn't seem like the desired behavior either.

Reply:
This is probably the #1 unresolved issue in this addition

Reply:
I called out something similar on the RFC-side, inlining my suggestions here too:

> Rather than finding `NetworkManager` with `GameObject.FindWithTag`, it might be useful to have a `static NetworkManager[]` array globally where we could quickly iterate over and find the one we're looking for.

> Similar to what I said above, we could store `NetworkManager` instances in a `static NetworkManager[]` container to avoid `Object.FindObjectOfType` call.

> _(response to Matt's Dictionary idea)_
> oh, static array or hashmap, the main idea being avoiding gameobject hierarchy traversal with `Object.FindObjectOfType` because seems like it could be unnecessary with a faster centralized static container lookup.

Reply:
Yeah, I think one way or another we'll want a central table of NMs